app/services/partners/request_approval_service.rb

```diff
@@ -27,7 +27,11 @@ def call
 
     def valid?
       if partner.profile.partner_status == 'submitted'
-        errors.add(:base, 'partner has already requested approval')
+        errors.add(:base, 'This partner has already requested approval.')
+      end
+
+      if !partner.profile.valid?
+        errors.add(:base, 'You must either provide a social media site or indicate that you have no social media presence before submitting for approval.')
```

Comment:
I would prefer to see a tighter check than partner.profile.valid?  here -- at least partly because I know there's another similar change coming down the pipeline where we'll need a different error for a different set of conditions.  Could we check for the condition where there is no social media and the "no social media" box isn't checked (and the organization uses the social media part of the card)?